### PR TITLE
Fix #25874 Add controls to stories

### LINKED ISSUE #25874: UI: Add controls to stories
### Parent Issue

_No response_

### Task

Create examples on how to add controls to the stories in Storybook

### Proposed Objective

Same as Parent Issue

### Proposed Priority

Priority 3 - Average

### Acceptance Criteria

Create examples on how to add controls to the stories in Storybook

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by fmontes**
### Summary
📝🛠️🏷️

This pull request updates and adds story files for various components using Storybook. The changes aim to improve the documentation, interactivity, and consistency of the component stories. The components affected include `DotCopyButtonComponent`, `Button`, and `AddStyleClassesDialogComponent`.

> _`DotCopyButtonComponent`_
> _New story file with tooltip_
> _Clipboard function_

### Walkthrough
*  Update story file for PrimeNG button component to use latest Storybook syntax and features, and add more options for button styles, sizes, icons, and positions ([link](https://github.com/dotCMS/core/pull/25875/files?diff=unified&w=0#diff-e9683a7beb56278068d4a40e151209d3cd29dc634a9533437b72b402702b6dddL1-R57))
* Add new story file for custom DotCopyButtonComponent that wraps PrimeNG button and adds tooltip and clipboard functionality ([link](https://github.com/dotCMS/core/pull/25875/files?diff=unified&w=0#diff-7d7f4ff3c884aaf898ea9d40db3809aaccd006e34b7559793e7fffeea3077decR1-R42))
* Change title of story file for custom AddStyleClassesDialogComponent to include `Library` prefix for better organization ([link](https://github.com/dotCMS/core/pull/25875/files?diff=unified&w=0#diff-ba61bc5a1858e46ace1419dbe92fdda53a3c0e659f1644b57c293bcc3d032901L26-R26))

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **

### CODE REVIEW

File: core-web/apps/dotcms-ui/src/stories/primeng/button/Button.stories.ts [hunk 1]
```diff
@@ -1,35 +1,57 @@
-import { moduleMetadata } from '@storybook/angular';
-import { Meta, Story } from '@storybook/angular/types-6-0';
+// How to write stories? https://storybook.js.org/docs/6.5/angular/writing-stories/introduction
+// Controls https://storybook.js.org/docs/6.5/angular/essentials/controls
+// Annotations: https://storybook.js.org/docs/6.5/angular/essentials/controls#annotation
```
Comment (zJaaal) [inline core-web/apps/dotcms-ui/src/stories/primeng/button/Button.stories.ts:3]: NIce!

**Comment by fmontes** [comment]
https://github.com/dotCMS/core/assets/751424/ee2c7d71-9e1c-4731-a0f4-21baafc3e129

File: core-web/apps/dotcms-ui/src/stories/primeng/button/Button.stories.ts [hunk 2]
```diff
@@ -1,35 +1,57 @@
...
+        size: 'p-button-md',
+        iconPos: 'left'
+    },
+    argTypes: {
+        size: {
+            options: ['p-button-sm', 'p-button-md', 'p-button-lg'],
+            control: { type: 'radio' }
+        },
+        severity: {
+            options: ['p-button-secondary', 'p-button-danger'],
```
Comment (zJaaal) [inline core-web/apps/dotcms-ui/src/stories/primeng/button/Button.stories.ts:25]: do we want a blank one to see the primary severity?

File: core-web/apps/dotcms-ui/src/stories/primeng/button/Button.stories.ts [hunk 3]
```diff
@@ -1,35 +1,57 @@
...
+            control: { type: 'select' }
+        },
+        type: {
+            options: ['p-button-text', 'p-button-outlined'],
```
Comment (zJaaal) [inline core-web/apps/dotcms-ui/src/stories/primeng/button/Button.stories.ts:29]: Same here

**Review by zJaaal — CHANGES_REQUESTED** [review]

**Review by zJaaal — APPROVED** [review]

**Review by rjvelazco — APPROVED** [review]

**Pull request merged**